src/events/messageCreate.ts

```diff
@@ -1,7 +1,6 @@
-import { embedMessage, event, Events, normalMessage, UserMessage } from '../utils/index.js'
-import { getChannelInfo, getServerConfig, getUserConfig, openChannelInfo, openConfig, UserConfig, getAttachmentData } from '../utils/index.js'
-import { clean } from '../utils/mentionClean.js'
 import { TextChannel } from 'discord.js'
+import { embedMessage, event, Events, normalMessage, UserMessage, clean } from '../utils/index.js'
+import { getChannelInfo, getServerConfig, getUserConfig, openChannelInfo, openConfig, UserConfig, getAttachmentData } from '../utils/index.js'
 
```

Comment:
Should we just import once? We can break it with a new line.

Reply:
I was trying to rectify that yesterday, does that meet the styling guidelines?

Reply:
As long as the application is working as intended. Nothing is wrong with it.